providers/common/ciphers/aes_basic.c

```diff
@@ -0,0 +1,966 @@
+/*
+ * Copyright 2001-2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/opensslconf.h>
+#include <openssl/crypto.h>
+#include <openssl/evp.h>
+#include <openssl/err.h>
+#include <string.h>
+#include <assert.h>
+#include <openssl/aes.h>
+#include "internal/evp_int.h"
+#include <openssl/rand.h>
+#include <openssl/cmac.h>
+#include "ciphers_locl.h"
+
+#define MAXBITCHUNK     ((size_t)1<<(sizeof(size_t)*8-4))
```

Comment:
A comment explaining this, at least the ```-4``` part, would be nice.  And add spaces around the operators.  (Yes I know this is moved code, but still...)

Reply:
This is copied code...not sure I'm in a position to write that comment.